### PR TITLE
gimme 1.6.0

### LINKED ISSUE #5: Update Homebrew source
The installer for Homebrew mentioned in the README needs to be updated to point to this repository.

https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/gimme.rb

### PR DESCRIPTION
**Pull request by meatballhat**
Created with `brew bump-formula-pr`.

Given [new home](https://github.com/travis-ci/gimme/issues/206) and [new release](https://github.com/urfave/gimme/releases/tag/v1.6.0).

I realize there are security and/or political implications here given I'm requesting the formula point at a different source. You should be able to see in the history at [travis-ci/gimme](https://github.com/travis-ci/gimme/commits?author=meatballhat) that I have been a core contributor since the beginning when the repository was at `meatballhat/gimme`. I'm happy to answer any questions around why I'm taking over maintenance again ([see relevant issue](https://github.com/travis-ci/gimme/issues/206)).

Closes urfave/gimme#5

### CODE REVIEW

**Comment by SMillerDev** [comment]
Is there any reference upstream announcing that you are the new maintainer?

**Review by carlocab — APPROVED** [review]
Makes sense to me; thanks for the update!

**Comment by iMichka** [comment]
Given there was no answer on https://github.com/travis-ci/gimme/issues/206 but it has only been 9 days, I would like to wait a little bit (let's say max 30 days).

It's a weird situation, and the explanation is really clear. If we get no answer, I'm fine with going ahead with the fork.

**Comment by meatballhat** [comment]
> Is there any reference upstream announcing that you are the new maintainer?

It's a bit weird, because I personally have been the maintainer for most of the project history, but I got laid off from Travis CI in 2019 after an acquisition. I'd like to get maintenance flowing again, and my attempts to work with current Travis CI maintainers haven't gone anywhere 😐 But, to @iMichka's point, I'm happy to wait a bit longer 😁

**Comment by MikeMcQuaid** [comment]
@meatballhat I sympathise with your position, that sounds rough ❤️. I feel like we need to hold off a migration until the "official" upstream approves it somewhere/somehow, just to avoid setting a precedent with this in future. Hopefully you understand we need to be overly cautious on this sort of thing 🙇🏻

**Comment by meatballhat** [comment]
@MikeMcQuaid Absolutely, I understand! Thank you ❤️

**Comment by iMichka** [comment]
I've asked upstream one more time for a clarification: https://github.com/travis-ci/gimme/issues/206#issuecomment-1345238598

**Comment by p-linnane** [comment]
Upstream is still silent on this. I think it is time for us to make a decision one way or another. How do we want to handle it?

**Comment by MikeMcQuaid** [comment]
@p-linnane If upstream is silent: we need to create a new formula with a new name and can consider deprecating this one.

**Comment by p-linnane** [comment]
@MikeMcQuaid Last commits were this past June, and v1.5.5 was cut then. Maybe we should wait a few more months on deprecation.

@meatballhat If you'd still like to add your fork, please open a PR for it with a new name.

**Pull request closed without merging**
